functions/zoom-downloader.py

```diff
@@ -222,13 +223,13 @@ def _class_schedule(self):
             Key={"zoom_series_id": str(self.data["zoom_series_id"])}
         )
 
-        if "Item" not in r:
-            return None
-        else:
+        if "Item" in r:
             schedule = r["Item"]
             # DynamoDB sometimes returns type decimal.Decimal
             schedule["opencast_series_id"] = str(schedule["opencast_series_id"])
             return schedule
+        else:
+            return None
```

Comment:
you made me google: https://stackoverflow.com/questions/4027586/is-it-ok-to-skip-return-none

personally, i prefer returning None early if some condition isn't met. Then, if I'm being mindful about it, i'd skip the `else:` entirely.  There's some airbnb eslint rule that actually complains when I do that in javascript. But whatever.

Reply:
Updated